pylint/utils/utils.py

```diff
@@ -253,6 +254,29 @@ def _check_csv(value: list[str] | tuple[str] | str) -> Sequence[str]:
     return _splitstrip(value)
 
 
+def _check_regexp_csv(value: list[str] | tuple[str] | str) -> Iterable[str]:
```

Comment:
Its not too clear what the function aims to achieve without scrutinising the body - should we add a docstring?